lang/es.po

```diff
@@ -9,6 +9,7 @@ msgstr ""
 # Put something like "Latin translation by Jvlivs Gaivs\n" here.
 msgid "TRANSLATION_AUTHOR\n"
 msgstr "Traducción al Español por Mr. Outis\n"
+msgstr "Traducción al Español por Natalie Stroud\n"
```

Comment:
This creates two alternative translations for the string. What I'd like you do is something like:
```
msgtr "Traducción al Español por Mr. Outis y Natalie Stroud\n"
```
I don't know Spanish, so no idea if one should use `y` or `e` or something else here.